src/BrazeKit-dev.js

```diff
@@ -965,6 +994,18 @@ var constructor = function () {
         }
     }
 
+    function decodeSubscriptionGroupMappings(subscriptionGroupSetting) {
+        let subscriptionGroupIds = {}; 
+      	const decodedSetting = subscriptionGroupSetting.replace(/&quot;/g, '"');
+        const parsedSetting = JSON.parse(decodedSetting)
+        for (let subscriptionGroupMap of parsedSetting) {
+            const key = subscriptionGroupMap.map
+            const value = subscriptionGroupMap.value
+            subscriptionGroupIds[key] = value
+        }
+        return subscriptionGroupIds
+    }
```

Comment:
As much as I want to, we should avoid using `const` and `let` in kits since we don't have transpiling set up.

```suggestion
        var subscriptionGroupIds = {}; 
      	var decodedSetting = subscriptionGroupSetting.replace(/&quot;/g, '"');
        var parsedSetting = JSON.parse(decodedSetting)
        for (let subscriptionGroupMap of parsedSetting) {
            var key = subscriptionGroupMap.map
            var value = subscriptionGroupMap.value
            subscriptionGroupIds[key] = value
        }
        return subscriptionGroupIds
    }
```